### ISSUE TITLE
about hyperparameter: learning rate about updating condenses samples

### ISSUE THREAD

**Issue by Alan-Qin** [GeorgeCazenavette/mtt-distillation#3]
Hello, George! First of all, I must say that this is very nice work.

I have some doubts about the used hyperparameter **lr_img** for updating condenses samples. It is not mentioned how to choose **lr_img** in the paper. Besides, I conduct the experiment about 10 images about each class for CIFAR-10 in terms of Table 6 and only obtain 58.50% accuracy. Should I modify other hyperparameters?

**Comment by GeorgeCazenavette** [comment]
The good news is that you don't have to re-train new experts in order to run the distillation experiment again :)

**Comment by Alan-Qin** [comment]
Thanks, George! It should attribute to the condensed image initialization problem. 

Thanks, again!

**Comment by GeorgeCazenavette** [comment]
Just curious, did you try running the experiment again with the new setting?

**Comment by Alan-Qin** [comment]
yes! good performance!  

Thanks, George!

When we choose the init as 'random', we may set **lr_lr** as 1e-5. Then we can get more consistent/stable results. 

^_^
